code/src/components/Action.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { executeAction } from '../reducers/game'
+import styled from 'styled-components'
```

Comment:
You should write it in this order:

import React, { useState } from 'react'
import { useSelector, useDispatch } from 'react-redux'
import styled from 'styled-components'

import { executeAction } from '../reducers/game'